### PR TITLE
Add Downloads page

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/tyk-developer-portal/customise/downloads.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Downloads"
+date: 2020-03-10T18:09:52Z
+menu:
+  main:
+    parent: "Customise"
+weight: 7
+---
+
+If you want to have files available for download from your portal you can add them to your `/tyk-dashboard/portal/portal-assets` directory.
```
Comment (buger) [inline tyk-docs/content/tyk-developer-portal/customise/downloads.md]: The folder name should be `/opt/tyk-dashboard/...`

Comment (marksou) [reply tyk-docs/content/tyk-developer-portal/customise/downloads.md]: Have updated.

**Comment by buger** [comment]
Do you think we need a new separate small page for this? 
Maybe it makes sense to embed it somewhere else, on the page about portal customization? (can be related to custom templates and custom CSS).

**Comment by marksou** [comment]
I did wonder about that. I'll move it.

**Pull request merged**